packages/quantic/force-app/main/default/lwc/quanticSort/quanticSort.js

```diff
@@ -131,6 +154,71 @@ export default class QuanticSort extends LightningElement {
     this.hasInitializationError = true;
   }
 
+  /**
+   * Sets the error when custom sort options have an invalid configuration.
+   */
+  setSortOptionsConfigurationError() {
+    this.errorMessage = `Custom sort options configuration is invalid.`;
+  }
+
+  /**
+   * Generates the schemas for the validation of the sort options with Bueno.
+   */
+  generateSchemas() {
+    this.headless = getHeadlessBundle(this.engineId);
+    const requiredNonEmptyString = new Bueno.StringValue({
+      required: true,
+      emptyAllowed: false,
+    });
```

Comment:
I would perhaps start creating a `validationUtils.js` file under the `quanticUtils/` folder.

And you're going to have to add this line to the `quanticUtils.js`:

`export * from 'c/validationUtils';`

That way in this file you should be able to `import {requiredNonEmptyString} from 'c/quanticUtils';` 

But I am not sure if it'll work due to the `new Bueno.StringValue` part here.

Reply:
yeah it doesnt seem to like it 😓

Reply:
What's the issue?

Reply:
it does not work because of the Bueno part, I will leave it as is since its just for 1 const.